releases/2022.2/2022.2.md

```diff
@@ -0,0 +1,171 @@
+
+In addition, kytos core now provides `asyncio` integration for event listeners with a new `alisten_to` decorator, so NApps can benefit from more lightweight handlers. Also, this open doors for future integration with `asyncio` libraries and its ecosystem.
+
+### 4. APM and logging infrastructure core integration
+
+Kytos core has been integrated with Elasticsearch [APM Python client](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP027.rst) to have performance and monitoring metrics. This [page](https://github.com/kytos-ng/kytos#how-to-use-with-elastic-apm) documents how to use and enable the APM client with Kytos. Elasticsearch, Filebeat, and Kibana can also be used in order to have a robust, indexed, and searchable logs infrastructure services based on [EP028 blueprint](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP028.rst). Figure 2 illustrates these services integrated with Kytos and a network operator using Kibana Web user interface to either query for logs or application performance metrics.
+
+<div align="center">
+  <a href="assets/elasticsearch_kibana.png"><img src="assets/elasticsearch_kibana.png"></a>
+</div>
+<div align="center">
+  Figure 2 - Elasticsearch, Kibana, Filebeat and APM integration with Kytos.
+</div>
+
+### 5. Link liveness detection
+
+`of_lldp` has been extended to be able to provide [link liveness detection](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP030.rst), leveraging the existing LLDP messages, in order to detect link transport outage that might not necessarily trigger a port down event. Link liveness detection can be enabled per interface and it needs to be enabled on each pair in order to work. This implementation is meant to provide a generic alternative to BFD (Bidirectional Forwarding Detection - RFC 5880) where the protocol states is being tracked by the control plane.
+
+### 6. Interface loop detection
+
+`of_lldp` has also been augmented to detect if an interface of a switch is looped. [Interface loop detection](https://github.com/kytos-ng/kytos/blob/master/docs/blueprints/EP025.rst) is automatically enabled once LLDP since it is been designed to detect unexpected or accidental loops that can happen. This feature has configuration options to configure configure which action should be applied once a loop is detected, by default, it generates a log warning message, but it can also disable an interface. Also, loops can be configured to be ignored.
+
+### 7. New Web UI components
+
+The following Web UI functionalities were added on `topology`:
```

Comment:
@rmotitsuki if you could also double check the UI features or the presented information that'd be great. Let me know if you'd like to highlight any other major objective or point.

Reply:
@viniarck It looks like all the main goals are listed.
Can you try replacing the Kytos UI screenshots with a 4:3 or 16:9 aspect ratio? The second image is too stretched.

Reply:
Right. Good idea @rmotitsuki, done here https://github.com/kytos-ng/kytos-ng.github.io/pull/34/commits/da2880b746725e698d8830a3e3904a65438cb3ca

Reply:
I'll mark this as resolved. Thanks.